modules/apps/commerce/commerce-product-options-web/src/main/resources/META-INF/resources/edit_cp_specification_option.jsp

```diff
@@ -53,4 +53,8 @@ else {
 			id="<%= CPSpecificationOptionFormNavigatorConstants.FORM_NAVIGATOR_ID_COMMERCE_PRODUCT_SPECIFICATION_OPTION %>"
 		/>
 	</div>
-</aui:form>
\ No newline at end of file
+</aui:form>
+
+<liferay-frontend:component
+	module="js/edit_cp_option_and_value"
+/>
```

Comment:
If you really need this, please make a copy paste of the file and name it **edit_cp_specification_option.js**